apps/prairielearn/src/ee/pages/instructorAiGenerateDrafts/instructorAiGenerateDrafts.html.ts

```diff
@@ -40,59 +87,216 @@ export function InstructorAIGenerateDrafts({
+                          <hr />
+
+                          <div class="mb-3">
+                            <label for="user-prompt-example" class="form-label">
+                              Or choose an example prompt:
+                            </label>
+                            <select
+                              id="user-prompt-example"
+                              onchange="setPromptToExample()"
+                              class="custom-select"
+                            >
+                              <option value=""></option>
+                              ${examplePrompts.map(
+                                (question) =>
+                                  html`<option
+                                    value="${question.id}"
+                                    data-prompt-general="${question.promptGeneral}"
+                                    data-prompt-user-input="${question.promptUserInput}"
+                                    data-prompt-grading="${question.promptGrading}"
+                                  >
+                                    ${question.id}
+                                  </option>`,
                               )}
-                        </td>
-                        <td>${row.uid ?? '(System)'}</td>
-                        <td>
-                          <a
-                            href="${resLocals.urlPrefix}/ai_generate_editor/${row.question_id}"
-                            class="btn btn-xs btn-primary"
-                          >
-                            Continue editing
-                          </a>
-                        </td>
-                      </tr>
-                    `,
-                  )}
-                </tbody>
-              </table>
+                            </select>
+                          </div>
+                        `
+                      : ''
+                  }
+
+                  <button class="btn btn-primary">
+                    <span
+                      class="spinner-grow spinner-grow-sm d-none"
+                      role="status"
+                      aria-hidden="true"
+                      data-loading-class-remove="d-none"
+                    >
+                    </span>
+                    Create question
+                  </button>
+
+                  <div
+                    id="loading-phrases"
+                    class="d-none mt-1"
+                    data-loading-class-remove="d-none"
+                  ></div>
+                  <div id="generation-results"></div>
+                </div>
+              </form>
+              ${hasDrafts ? html`<div class="reveal-fade"></div>` : ''}
             </div>
+            ${hasDrafts
+              ? html`
+                  <div
+                    class="p-2 d-flex justify-content-center bg-light js-expand-button-container"
+                  >
+                    <button type="button" class="btn btn-sm btn-link">Expand</button>
```

Comment:
The "Expand" button doesn't work for me in Safari. It does work in Chrome. Safari does expand when clicking inside the form.

Reply:
Good reminder to do cross-browser testing. I can reproduce that too. I also tried it in Firefox and the styling is all messed up on the fade part:

<img width="719" alt="Screenshot 2025-01-26 at 20 33 22" src="https://github.com/user-attachments/assets/50e596a4-5945-4c84-b8dc-c55b8362ebf8" />

Reply:
This is wild, Safari doesn't seem to be firing/propagating the `click` event on the button, even when it's clicked??

Reply:
I fixed the Safari issue in 68410d5. I guess `querySelector` returns different elements in different browsers? At any rate, it may as well have been a bug with my code, I had specified a less-precise container element than I could have.

Reply:
Yuck, I somehow thought the state of cross-browser compatibility was better in 2025 😭